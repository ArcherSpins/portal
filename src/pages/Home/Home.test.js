// TODO: TDD
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import React from 'react';

import { Home } from '.';

const component = <Home />;
describe('The Home component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(component);
    expect(wrapper.hasClass('home')).toBeTruthy();
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
