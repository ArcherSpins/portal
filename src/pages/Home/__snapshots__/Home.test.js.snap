// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home component renders correctly 1`] = `
<div
  className="home"
>
  Hello World!
</div>
`;
